src/main/scala/io/viash/functionality/dependencies/Repository.scala

```diff
@@ -62,14 +63,22 @@ object Repository {
     }
   }
 
-  def cache(repo: Repository): Repository = 
+  def cache(repo: Repository, configPath: Path, projectRootDir: Option[Path]): Repository = 
     repo match {
       case r: GithubRepository => {
         val r2 = r.checkoutSparse()
         r2.checkout()
       }
       case r: LocalRepository if r.path.isDefined => {
-        r.copyRepo(localPath = r.path.get)
+        var localPath = r.path.get match {
```

Comment:
Why `var`? Is a `val` not sufficient here?

```suggestion
        val localPath = r.path.get match {
```

Reply:
Maybe a typo (R and L are right next eachother on Dvorak) or otherwise a brain fart?